src/components/admin/app-navbar/AppNavbar.vue

```diff
@@ -94,6 +94,11 @@ export default {
 </script>
 <style lang="scss">
 .app-navbar {
+  position: fixed !important;
```

Comment:
Maybe makes more sense to move this to vuestic-ui.

Reply:
fixed